.travis.yml

```diff
@@ -29,7 +29,12 @@ env:
   - DOCKER_DIR="../docker/"
 
 script:
-  - sbt ci
+  - export TEST_SKIP_REGEX="\[skip-tests\]"
+  - if [[ (! $TRAVIS_COMMIT_MESSAGE =~ $TEST_SKIP_REGEX) || -n $TRAVIS_TAG || $TRAVIS_BRANCH == "master" ]];
```

Comment:
Shouldn't we only skip tests if the commit message is set?  This conditional implies that we know all the cases in which the tests should run